extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/api/RegistrationService.java

```diff
@@ -46,6 +66,36 @@ public List<Participant> listParticipants() {
 
     public void addParticipant(Participant participant) {
         monitor.info("Adding a participant in the dataspace.");
-        participantStore.addParticipant(participant);
+        participantStore.save(participant);
+    }
+
+    public void start() {
```

Comment:
Let's either add javadocs to `start` `stop` and `addParticipant` methods or remove one from `listParticipants()` to be consistent